content/en/developers/_index.md

```diff
@@ -12,13 +12,13 @@ further_reading:
 
 ## Overview
 
-The Developers section contains reference materials for developing for Datadog.
+The Developers section contains reference materials for developing on Datadog. You may want to develop on Datadog if there is data you want to see in the product that you are currently not seeing, or if you are a partner who knows they want to build on Datadog and contribute it to the Datadog Marketplace or to Integrations Extras. For more information about becoming a Datadog partner, see our [Partner program page][1].
```

Comment:
```suggestion
The Developers section contains reference materials for developing on Datadog. You may want to develop on Datadog if there is data you want to see in the product that you are not seeing, or if you are a partner who wants to build on Datadog and contribute to the Datadog Marketplace or to Datadog's community integrations. For more information about becoming a Datadog partner, see the [Partner program page][1].
```

I'm feeling that "integrations extras" is an internal name and that normal people probably just think of them as "Datadog's community integrations"?